src/Common/Core/Impl/Imaging/IImageService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Imaging {
+    /// <summary>
+    /// Provides access to images and icons
+    /// </summary>
+    public interface IImageService {
```

Comment:
There is preliminary split of Components in my Components branch. Waiting for @karthiknadig to complete Host.Client split since there are Shiny etc supporting code that uses objects not available in .NET Core